domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/service/InductionScheduleStatusTransitionValidator.kt

```diff
@@ -41,12 +41,12 @@ class InductionScheduleStatusTransitionValidator {
       reason = "Cannot transition from COMPLETED to any other status.",
     ),
     ValidationRule(
-      condition = { current, new -> current.isExemptionOrExclusion() && new.isExemptionOrExclusion() },
-      reason = "Cannot transition from one exemption or exclusion status to another.",
+      condition = { current, _ -> current == InductionScheduleStatus.PENDING_INITIAL_SCREENING_AND_ASSESSMENTS_FROM_CURIOUS },
+      reason = "Cannot transition from PENDING_INITIAL_SCREENING_AND_ASSESSMENTS_FROM_CURIOUS to any other status.",
     ),
     ValidationRule(
-      condition = { current, new -> new == InductionScheduleStatus.SCHEDULED && !current.isExemptionOrExclusion() },
```

Comment:
Removed this validation rule (for InductionSchedule status changes) because when re-scheduling an Induction Schedule we _do_ want to set it's status to SCHEDULED